modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ImportTranslationResultsDisplayContext.java

```diff
@@ -66,6 +67,26 @@ public int getFailureMessagesCount() {
 		return _failureMessages.size();
 	}
 
+	public String getFailureMessagesCSV(Locale locale) {
+		StringBuilder sb = new StringBuilder();
```

Comment:
Just a note for your future self: in general, we use the `StringBundler` class for concatenation, not `StringBuilder`.